### PR TITLE
Add sound refactor to use prefabs

### PR DESCRIPTION
**Pull request by allenwhitedev**
Add sound effects to game: player hurt, zombie hurt, zombie die, zombie idle, pistol shoot, pistol reload, weapon switch
only allow Player prefab to shoot if user has ammo

### CODE REVIEW

File: Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs [hunk 1]
```diff
@@ -44,7 +45,7 @@ void Update()
             muzzleFlashTimer += Time.deltaTime;
         }
         coolDownTimer += Time.deltaTime;
-        cooldownBar.SetFill(coolDownTimer, currGun.cooldownTime);
+        // cooldownBar.SetFill(coolDownTimer, currGun.cooldownTime);
```
Comment (danielschneider22) [inline Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs]: Is this in final commit? I want cooldown bar to be in playergeneralshootcontroller for main game

File: Assets/Scripts/PlayerShoot/PlayerShootController.cs [hunk 2]
```diff
@@ -17,7 +17,11 @@ private void Awake()
     // Update is called once per frame
     void Update()
     {
-        if(Input.GetButton("Fire1") && !playerGeneralShootController.IsReloading() && playerGeneralShootController.coolDownTimer > playerGeneralShootController.currGun.cooldownTime)
+			Debug.Log("PlayerShoot Update");
```
Comment (danielschneider22) [inline Assets/Scripts/PlayerShoot/PlayerShootController.cs:20]: Remove debug log

File: Assets/Scripts/MainMenuManager/PlayClick.cs [hunk 3]
```diff
@@ -14,17 +14,10 @@ public class PlayClick : MonoBehaviour
...
-       // muzzleFlash.enabled = false; 
-    }
+    void Start() {}
 
     // Update is called once per frame
-    void Update()
-    {
-        
-    }
+    void Update() {}
```
Comment (danielschneider22) [inline Assets/Scripts/MainMenuManager/PlayClick.cs:22]: You can actually just fully remove start and update if you aren't using them

File: Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs [hunk 4]
```diff
@@ -68,18 +70,18 @@ public void ShootHandgun()
         rb.AddForce(transform.right * currGun.bulletSpeed, ForceMode2D.Impulse);
         numBulletsInGun--;
         CorrectAmmoText();
-        if (numBulletsInGun <= 0)
-        {
-            Reload();
-        }
-
+				audioManager.Play("PistolShoot", 0);
```
Comment (danielschneider22) [inline Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs]: Phweeeee nice!

File: Assets/Scripts/Rooms/MapLayoutManagerStepThrough.cs [hunk 5]
```diff
@@ -0,0 +1,441 @@
+﻿using System.Collections;
```
Comment (danielschneider22) [inline Assets/Scripts/Rooms/MapLayoutManagerStepThrough.cs]: This looks like code I made?

Comment (allenwhitedev) [reply Assets/Scripts/Rooms/MapLayoutManagerStepThrough.cs]: yea something beyond my git comprehension is happening,
I git pulled the 'main' branch into this branch using the "ours" recursive strategy

File: Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs (hunk 1 above)
Comment (allenwhitedev) [reply Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs]: commented this coolDownBar.setFill code back in, not sure how/why it was commented-out

File: Assets/Scripts/MainMenuManager/PlayClick.cs (hunk 3 above)
Comment (allenwhitedev) [reply Assets/Scripts/MainMenuManager/PlayClick.cs:22]: removed

File: Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs (hunk 4 above)
Comment (allenwhitedev) [reply Assets/Scripts/PlayerShoot/PlayerGeneralShootController.cs]: pew pew

**Comment by allenwhitedev** [comment]
recreated changes on new Pull Request with unity version update included https://github.com/danielschneider22/Stealth-Shooter/pull/9

**Pull request closed without merging**